library/psa_crypto.c

```diff
@@ -4695,6 +4695,129 @@ static void psa_des_set_key_parity( uint8_t *data, size_t data_size )
 }
 #endif /* MBEDTLS_PSA_BUILTIN_KEY_TYPE_DES */
 
+/*
+* ECC key types require the generation of a private key which is an integer
```

Comment:
Note that this is just the method for Weierstrass curves. Montgomery curves use a different method.

Reply:
Fixed